sdk/attestation/attestation/test/utils/recordedClient.ts

```diff
@@ -23,7 +23,7 @@ import "./env";
 const replaceableVariables: { [k: string]: string } = {
   AZURE_CLIENT_ID: "azure_client_id",
   AZURE_CLIENT_SECRET: "azure_client_secret",
-  AZURE_TENANT_ID: "azure_tenant_id",
+  AZURE_TENANT_ID: "12345678-1234-1234-1234-123456789012",
```

Comment:
The dummy tenant ID should be azure_tenant_id, IMHO.

Reply:
I updated the description. We have updated the identity version, the new MSAL doesn't allow us to use a dummy tenant id, it does format checks, so we need a dummy tenant id which looks more like a real tenant id.

Reply:
Ah, that makes much more sense then.

Reply:
@LarryOsterman  Can you re-record the tests and update the recordings for attestation package?